### PR TITLE
[IE TESTS MYRIAD] Enabling of NonZero single layer tests

### PR DESCRIPTION
**Pull request by rvyunov**
#31090

### CODE REVIEW

**Comment by rvyunov** [comment]
@AndrewBakalinIntel , @Maxim-Doronin , could you review it, please?

File: inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/nonzero.cpp [hunk 1]
```diff
@@ -16,10 +16,25 @@ using namespace LayerTestsDefinitions;
 
 namespace {
 
+class MyriadNonZeroLayerTest : public NonZeroLayerTest {
+protected:
+    void SetUp() override {
+        configuration[VPU_CONFIG_KEY(DETECT_NETWORK_BATCH)] = CONFIG_VALUE(NO);
```
Comment (Maxim-Doronin) [inline inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/nonzero.cpp:22]: I think you can modify NonZeroLayerTest to pass `ConfigMap` like it has been done in Eltwise tests.
Thus there be no need to create heir class

Comment (rvyunov) [reply inference-engine/tests/functional/plugin/myriad/shared_tests_instances/single_layer_tests/nonzero.cpp]: Good catch! Thx. It fixed.

**Comment by rvyunov** [comment]
@Maxim-Doronin, CI has been passed. Could you merge it, please?

**Pull request merged**